job.go

```diff
@@ -97,3 +161,60 @@ func (jl JobList) Sort(sorter func(i, j int) bool) JobList {
 
 	return jl
 }
+
+//DoesJobContainTaskRange evaluates whether the XML marshalled tasks value contains the regex indicating a sequence of tasks.
+func DoesJobContainTaskRange(j Job) (bool, error) {
+	regex, err := regexp.Compile(TASKRANGEIDENTIFIERREGEX)
```

Comment:
wouldn't it make sense to move this compilation out of the loop - can we compile as a var?

var/const taskRangeRegex = regexp.MustCompile(TASSKRANGE...)

Reply:
It would make more sense. Totally forgot about it when I was going back and forth with the structures and marshaling.